sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/objects/objects.scala

```diff
@@ -236,7 +236,32 @@ case class StaticInvoke(
   override def children: Seq[Expression] = arguments
 
   lazy val argClasses = ScalaReflection.expressionJavaClasses(arguments)
-  @transient lazy val method = cls.getDeclaredMethod(functionName, argClasses : _*)
+  @transient lazy val method = {
```

Comment:
This is duplicated with `Invoke`. But this will be backported, so I will leave the merging of `method` only to master branch.

Reply:
Even though it's for backporting, shall we still avoid duplicated code and add a `def findMethod(cls: Class[_])` in the base class?

Reply:
ok